regext/charter.md

```diff
@@ -1,41 +1,23 @@
-
-Extensions to both protocols that are targeted for the IETF consensus are subject to more thorough review and open discussion within the IETF. 
-In addition, commonality may be discovered in related extensions for which it would make sense to merge them into a single standard extension everybody agrees on.
-
-The REGEXT working group is the home of the coordination effort for EPP and RDAP extensions. 
-The selection of extensions shall incorporate the following guidelines:
-
-Proprietary documented extensions and individual submissions will follow the expert review process as described in the appropriate extensions registry. 
-These extension documents will not be part of the REGEXT working group work or milestones. 
-The working group may discuss or advise on these documents.
 
-* Extensions that seek IETF consensus can be suggested for WG adoption. 
-  If accepted by the working group then the development of the extension may proceed.
+The working group may publish proposed standard and experimental extensions to EPP and RDAP.
+The working group may publish informational and best current practice documents for EPP and RDAP.
+The working group may revise the registration procedures for the EPP and RDAP registries.
+The working group will discuss new registrations with the designated experts for the EPP and RDAP registries.
+When the working group is closed, the mailing list will remain open and available for the use of the expert review process.
 
-* When there are no more proposals for IETF consensus extensions, the working group will either close or become dormant, with the decision made in consultation with the responsible AD. 
-  In any case, the mailing list will remain open and available for the use of the expert review process.
+A primary goal of the working group is to ensure the profiles and extensions are easily discoverable and understood, and to avoid duplicate effort that could harm interoperability.
 
-The working group may also take on work to develop specifications that are related to the EPP or RDAP registration processes, describe the information exchanged between entities involved in Internet identifier registration that are using the RDAP or EPP protocols. 
-This decision will be made in consultation with the responsible AD.  
-Possible work includes, but is not limited to the following topics:
+In addition to publishing extensions, the working group may publish proposed standards addressing interoperability issues that are adjacent to EPP and RDAP.
 
-* Uniform representation formats for publishing local policy or configuration options regarding EPP and RDAP use.
+For example:
 
-* Data formats for files exchanged between registration entities that need insertion in or extraction from EPP or RDAP.
+* Data formats for exchanges between registration entities that need insertion in or extraction from EPP or RDAP.
```

Comment:
The phrase "... insertion in or extraction from EPP or RDAP" reads weird since EPP and RDAP are not databases. Since EPP is a read-write protocol and RDAP a read protocol, how about replacing that phrase with something like "... insertion using EPP, or extraction using EPP or RDAP"?